internal/terraform/ast/variables.go

```diff
@@ -52,6 +52,14 @@ func VarsFilesFromMap(m map[string]*hcl.File) VarsFiles {
 	return mf
 }
 
+func (mf VarsFiles) Copy() VarsFiles {
+	m := make(VarsFiles, len(mf))
+	for name, file := range mf {
```

Comment:
```suggestion
func (vf VarsFiles) Copy() VarsFiles {
	m := make(VarsFiles, len(vf))
	for name, file := range vf {
```
Nitpick: Even though we use short names they can still have some meaning, in this case `vf` = vars files 😉